### ISSUE TITLE
SuperPWA break batch plugins activation

### ISSUE THREAD

**Issue by ariona** [SuperPWA/Super-Progressive-Web-Apps#93]
Hello, I am a theme developer and I included the SuperPWA in my required plugins using TGMPA. Installing the plugin is no problem, but when activating the plugins at the same time (batch action), the process stop on PWA with the error:

```
Fatal error: Uncaught Error: Class 'WP_Plugins_List_Table' not found in /Applications/MAMP/htdocs/wp-sandbox/wp-content/plugins/super-progressive-web-apps/admin/basic-setup.php:74 
Stack trace: 
#0 /Applications/MAMP/htdocs/wp-sandbox/wp-includes/class-wp-hook.php(286): superpwa_activation_redirect('super-progressi...', false) 
#1 /Applications/MAMP/htdocs/wp-sandbox/wp-includes/class-wp-hook.php(310): WP_Hook->apply_filters('', Array) 
#2 /Applications/MAMP/htdocs/wp-sandbox/wp-includes/plugin.php(453): WP_Hook->do_action(Array) 
#3 /Applications/MAMP/htdocs/wp-sandbox/wp-admin/includes/plugin.php(625): do_action('activated_plugi...', 'super-progressi...', false) 
#4 /Applications/MAMP/htdocs/wp-sandbox/wp-admin/includes/plugin.php(759): activate_plugin('super-progressi...', '', false, false) 
#5 /Applications/MAMP/htdocs/wp-sandbox/wp-content/themes/librarium/inc/TGMPA/class-tgm-plugin-activation.php(3025): activate_plugins(Array) 
#6 /Applications/MAMP/htdocs/wp-sandbox/wp-content/themes/librarium/inc/TGMPA/class-tgm-plugin-ac in /Applications/MAMP/htdocs/wp-sandbox/wp-content/plugins/super-progressive-web-apps/admin/basic-setup.php on line 74
```

I am not sure if this is caused by TGMPA or SuperPWA plugin, but i hope the log above can give you guy a hint.

thanks

**Comment by arunbasillal** [comment]
Hello @ariona 

Thanks for the report. 

The class `WP_Plugins_List_Table` probably isn't loaded at the time. 

I have added a check in 2.1 (next release) so that this doesn't happen - https://github.com/SuperPWA/Super-Progressive-Web-Apps/commit/c70bfd96e415e4d8beee54b217b1290d70c553dd 

That should take care of it :)

**Comment by madmath03** [comment]
Hello @arunbasillal,

I encountered the same issue and this blocks my batch installs.
When is the next release planned to be deployed ?

**Comment by arunbasillal** [comment]
@madmath03 Planning it for the second week of March.

**Comment by madmath03** [comment]
It's been 2 months now... is the next release still coming ?

**Comment by madmath03** [comment]
@arunbasillal Is this plugin still maintained ??

**Comment by madmath03** [comment]
> @arunbasillal Is this plugin still maintained ??

Well, I guess not 😅

**Comment by bobmulder** [comment]
I see a new version was released but I guess it didn't solve this issue. Would like to get some response from the maintainers :)

**Comment by arunbasillal** [comment]
This should be fixed in 2.1 - https://github.com/SuperPWA/Super-Progressive-Web-Apps/commit/c70bfd96e415e4d8beee54b217b1290d70c553dd
